suricata.yaml.in

```diff
@@ -1991,6 +1991,11 @@ napatech:
     #
     hashmode: hash5tuplesorted
 
+security:
```

Comment:
after merging landlock a security config already exists that this should hook into